keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+    * Determine what phase it missed
+    * Identify multiple slips / misses
+
+#### Enhancements Subteam
+
+* Pre enhancements Freeze (Enhancements Collection)
+  * Collect intentions/plans of KEP authors to graduate a KEP
+  * Ensure that those intents comply with all requirements of the KEP
+  * Mark enhancements "At Risk" based on compliance with all requirements of the KEP
+
+* At Enhancements Freeze
+  * Remove enhancements that do not comply with requirements
+
+* Enhancements Freeze - Code Freeze
+  * Ensure that the k/k PRs for the KEP are populated
+  * Mark enhancements "At Risk" based on the progress of implementation
+  * Approve/Deny exceptions for Enhancements Freeze
+
+* Around Code Freeze
+  * Ensure that the enlisted k/k are merged
+  * Approve/Deny exceptions for Code Freeze
+
+* At Code Freeze
+  * Remove enhancements that have not completed requirements for Code Freeze
+
+* Throughout the Cycle
+  * Prepare a manifest of what's the status of enhancements at any point in time
+
+#### Docs Subteam
+
+* Enhancements Freeze - Open Docs PR Deadline
+  * Ensure that the k/w PRs are populated in the KEPs
+  * Communicate to SIG Docs about the expected review load
+
+* Open Docs PR Deadline - Docs Complete
+  * Mark enhancements "At Risk" if the following are true:
+    *  Docs PRs are not updated
+    *  The draft PRs have not been updated
+
+### Notes/Constraints/Caveats
+
+This KEP is based on a [high-level plan](https://docs.google.com/document/d/1qnfXjQCBrikbbu9F38hdzWEo5ZDBd57Cqi0wG0V6aGc/edit#) incorporating feedback collected from release team members and contributors over several cycles.
+
+In 1.22, we'll focus on implementing much of this process manually, so there will be some burden placed on KEP Authors to create the appropriate YAML structure. But ideally, these changes will free up time for people on the release team to work on automating the receipts process.
+
+### Risks and Mitigations
+
+* Non-compliance to the process by stakeholders like SIGs/KEP Authors
+  * Stricter gates have to be put for the process
+  * Exceptions only approach for a receipt to be added back to release cycle stage after the deadline
+  * Enhancement Team + Release Leads should have the final say on exceptions. The decision needs to be taken after considering the point of view of respective SIG leadership.
+* Conflicts can arise when two people update/modify the same artifact
+  * The team needs to have Git experience
+  * Only one person works on the artifacts at a time
+  * Communicate with each other when making changes
+  * Bots can eventually handle the manifest generation
+* Keeping KEP implementation history updated
+  * Requires intervention from a human and trust needs to be on the same person
+* Preventing enhancements to slip across release cycles
+* Ensuring enhancements don't permanently remain in the Beta stage
+* This KEP doesn't talk about KEP owners keeping enhancements issues up-to-date
```

Comment:
I guess I'm wondering what function does the issue serve? Will they be eliminated? This seems worthwhile to address here since it's part of the existing process